docs/components/index.md

```diff
@@ -0,0 +1,24 @@
+
+There are currently two major groups of endpoints in the API:
+
+
+<div class="grid cards"  markdown>
+
+- :material-file-document-multiple:{ .lg .middle } __Document Management__
+
+    ---
+
+    Upload and manage documents to be used by the AI assistant
+
+    [:octicons-arrow-right-24: More info](./doc-management.md)
+
+- :octicons-discussion-closed-16:{ .lg .middle .red} __The Question-Answering Service__
```

Comment:
Nit: We should be consistent in how we describe components/ services and drop the "The" here, since we don't have "The Document Management Service". Should just be "Document Management", "Question-Answering" etc.

Reply:
Also matches how they appear in the left of the screen under index. 
![Uploading Screenshot 2024-11-20 at 13.28.00.png…]()